src/resources/sparkConf.properties

```diff
@@ -68,10 +68,31 @@ spark.query.types                                 9,1,4,3
 
 # The tool adds TTL & Writetime at row-level (not field-level).
 # The largest TTL & Writetime values are used if multiple indexes are listed (comma separated)
-# Comma separated column indexes from "spark.query.origin" used to find largest TTL or Writetime
+# Comma separated column indexes from "spark.query.origin" used to find largest TTL or Writetime. This is zero-based index which starts at 0
 spark.query.ttl.cols                              2,3
 spark.query.writetime.cols                        2,3
 
+############################### EXAMPLE MAPPING USING A DEMO TABLE ##########################################
+# If the origin and target clusters have a schema such as below,
+# CREATE TABLE cycling.cyclist_name (
+#     pk1 uuid,
+#     pk2 date,
+#     cc1 boolean,
+#     firstname text,
+#     lastname text,
+#     phones list<text>,
+#     PRIMARY KEY((pk1,pk2),cc1)
+# );
+# 
+# then, our mapping would look like below,
+# spark.query.origin                                pk1,pk2,cc1,firstname,lastname,phones
+# spark.query.origin.partitionKey                   pk1,pk2
+# spark.query.target.id                             pk1,pk2,cc1
+# spark.query.types                                 9,15,10,0,0,6%0
+# spark.query.ttl.cols                              0,1,2
```

Comment:
spark.query.ttl.cols 3,4
park.query.writetime.cols 3,4

this is because, primary-key columns do not have ttl, writetimes. And we do not use collections columns as well for pulling TTL, writetimes

Reply:
why not `2,3,4` here? 🤔